web/profiles/custom/yalesites_profile/config/sync/core.entity_view_display.node.profile.default.yml

```diff
@@ -99,6 +97,7 @@ third_party_settings:
       - 'Content fields'
       - 'Custom block types'
       - 'Custom blocks'
+      - Devel
```

Comment:
I noticed Devel is in this file listed as a possible block that can be added? Is this OK or should be removed? @nJim

Reply:
Removing this line. Good catch.